gallery/src/components/NxFormSelect/NxFormSelectPage.tsx

```diff
@@ -132,6 +134,13 @@ const NxFormSelectPage = () =>
                         liveExample={NxFormSelectOverflowExample}>
       Demonstrates an <NxCode>NxFormSelect</NxCode> with long option text that overflows.
     </GalleryExampleTile>
+
+    <GalleryExampleTile title="Form Select Widths Examples"
```

Comment:
This should probably be on the `.nx-form-select` styles page, not the `NxFormSelect` component page.  Additionally the classes need to be called out in documentation, not just demonstrated in an example

Reply:
> This should probably be on the `.nx-form-select` styles page, not the `NxFormSelect` component page. 

Why only there? I can see wanting on both pages but why only `nx-form-select`? I would expect that most consumers would use the React component.

Reply:
> Why only there? I can see wanting it on both pages but why only nx-form-select? I would expect that most consumers would use the React component.

I'm ok with both but we generally only document styling variations on the style page in cases where it exists

Reply:
> This should probably be on the `.nx-form-select` styles page, not the `NxFormSelect` component page. Additionally the classes need to be called out in documentation, not just demonstrated in an example

Example and docs: https://github.com/sonatype/sonatype-react-shared-components/pull/586/commits/b72e1df55c137eb4a5cc1163ceacb20217ca10e1